app/portainer/components/DetailsTable/DetailsTable.tsx

```diff
@@ -0,0 +1,27 @@
+import { PropsWithChildren } from 'react';
+
+type Props = {
+  headers?: string[];
+  dataCy?: string;
+};
+
+export function DetailsTable({
+  headers,
+  dataCy,
+  children,
+}: PropsWithChildren<Props>) {
+  return (
+    <table className="table" data-cy={dataCy}>
+      {headers && headers.length > 0 && (
```

Comment:
instead of checking if it's defined, just give the headers a default value (see my comment)

Reply:
updated